File: JavaScript/Advanced/Lab-Modules/01.Person/person.js
/**
 * Created by tangrila on 04-Nov-16.
 */
let Person = function() {
	class Person {

		constructor(name) {
			this.name = name;
		}

		toString() {
			return `I'm ${this.name}`;
		}
	}

	return {
		Person
	}
}();

File: Csharp/OOP/Basics/DefiningClasses/03.OldestFamilyMember/Person.cs
namespace Csharp.OOP.Basics.DefiningClasses._03.OldestFamilyMember
{
    public static partial class _03OldestFamilyMember
    {
        private class Person
        {
            private string _name;
            private int _age;

            public int Age
            {
                get { return _age; }
                private set { _age = value; }
            }

            public override string ToString()
            {
                return $"{this._name} {this.Age}";
            }

            public Person(string name, int age)
            {
                this._name = name;
                this.Age = age;
            }
        }
    }
}
